ui/app/src/utils/i18n-legacy.ts

```diff
@@ -413,6 +413,15 @@ export const contentTypesMessages = defineMessages({
   toolNotFound: {
     id: 'siteConfig.toolNotFound',
     defaultMessage: '"{tool}" tool not found.'
+  },
+  dependsOn: {
+    id: 'contentType.dependsOn',
+    defaultMessage: 'This property depends on "{dependency}"'
+  },
+  dependencyNotImplemented: {
+    id: 'contentType.dependencyNotImplemented',
+    defaultMessage:
+      '[{property}.js] Control dependency not implemented for input of type "{property}". Dependency will be ignored.'
   }
```

Comment:
This was really just meant to be a console.log. No need for translation or UI popup etc. The file name should be fixed to the file name that issues the log so when we see it we can easily find, inspect and address if needed.